clients/privacy-center/components/consent/notice-driven/NoticeDrivenConsent.tsx

```diff
@@ -100,30 +72,38 @@ const NoticeDrivenConsent = ({ base64Cookie }: { base64Cookie: boolean }) => {
     return identities ? { ...deviceIdentity, ...identities } : deviceIdentity;
   }, [fidesUserDeviceId]);
 
+  // Full list of privacy notices, including children of other notices
+  const flatPrivacyNoticesList: PrivacyNoticeResponse[] = useMemo(() => {
```

Comment:
excellent way to rework so that we can use all our existing code for saving preferences 👍